### PR TITLE
EZEE-3020: Close tooltips when item is deleted

### PR DESCRIPTION
**Pull request by lucasOsti**
Question      | Answer
| ------------- | ---
| Tickets       | https://jira.ez.no/browse/EZEE-3020
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | no
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

Added auto hide of tooltips if tooltip node has been deleted

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js [hunk 1]
```diff
@@ -1,5 +1,17 @@
-(function(global, doc, eZ, $) {
+(function (global, doc, eZ, $) {
     const TOOLTIPS_SELECTOR = '[title]';
+    const config = {
+        attributes: true,
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js:4]: Correct me if I'm wrong but I think we don't have to observe the `attributes`?

Comment (lucasOsti) [reply src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js]: yup, you have right

File: src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js [hunk 2]
```diff
@@ -1,5 +1,17 @@
-(function(global, doc, eZ, $) {
+(function (global, doc, eZ, $) {
     const TOOLTIPS_SELECTOR = '[title]';
+    const config = {
+        attributes: false,
```
Comment (dew326) [inline src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js:4]: I think if it is false you can omit the option

File: src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js [hunk 3]
```diff
@@ -1,5 +1,17 @@
...
+        childList: true,
+        subtree: true,
+    };
+    const observer = new MutationObserver((mutationsList, observer) => {
+        mutationsList.forEach((mutation) => {
+            if (mutation.removedNodes.length && doc.querySelector('.ez-tooltip.show')) {
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js:10]: `doc.querySelector('.ez-tooltip.show')` could be put into variable, as it's used two times

File: src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js [hunk 4]
```diff
@@ -1,5 +1,17 @@
-(function(global, doc, eZ, $) {
+(function (global, doc, eZ, $) {
     const TOOLTIPS_SELECTOR = '[title]';
+    const config = {
+        attributes: true,
+        childList: true,
+        subtree: true,
+    };
+    const observer = new MutationObserver((mutationsList, observer) => {
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js:8]: Unnecessary second arg `observer` (could be even misleading, as variable `observer` is already defined)

File: src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js [hunk 5]
```diff
@@ -1,5 +1,18 @@
-(function(global, doc, eZ, $) {
+(function (global, doc, eZ, $) {
     const TOOLTIPS_SELECTOR = '[title]';
+    const config = {
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js:3]: Change to `observerConfig` , so it would be more specific

**Pull request merged**